packages/terra-clinical-data-grid/src/proptypes/columnDataShape.js

```diff
@@ -27,13 +32,13 @@ const columnDataShape = PropTypes.shape({
   text: PropTypes.string,
   /**
    * A string indicating which sorting indicator should be rendered. If not provided, no sorting indicator will be rendered.
-   * If a `component` value is specified, `sortIndicator` will be ignored.
+   * If a `component` value is specified, `sortIndicator` will be ignored. One of `ascending`, `descending`.
    */
-  sortIndicator: PropTypes.oneOf(['ascending', 'descending']),
+  sortIndicator: PropTypes.oneOf([SortIndicators.ASCENDING, SortIndicators.DESCENDING]),
```

Comment:
Could just do: 
```suggestion
  sortIndicator: PropTypes.oneOf(Object.values(SortIndicators)),
```

Reply:
Done in 4d40b4f2dccf8b009188b4a36fe230fd216d42c2